website/static/js/comment.js

```diff
@@ -298,7 +298,12 @@ var CommentModel = function(data, $parent, $root) {
             'gravatarUrl': userData.links.profile_image
         };
     } else {
-        self.author = self.$root.author;
+        self.author = {
+            'id': null,
+            'urls': {'profile': ''},
+            'fullname': 'Deactivated user',
```

Comment:
Although, given that we're not changing the name as it appears on the contributors list, perhaps we should not be changing it elsewhere, either. Or at most changing it to something like 
`'{} (deactivated)'.format(userData.attributes.full_name)`